app/main.py

```diff
@@ -40,7 +40,7 @@ async def send_edit_link(email: Email, background_tasks: BackgroundTasks):
         "We received your email address and will be sending you an edit"
         " link shortly!"
     )
-    email_address = email.email_address
+    email_address = email.email_address.lower()
```

Comment:
Might want to lower emails from the our data too?

Reply:
I think we end up doing that here: https://github.com/WATonomous/user-config-lookup/blob/0d87a9151b572287bdb2ba77fc924ddb0b00882f/app/utils.py#L28